internal/runner/rhel.go

```diff
@@ -13,7 +13,8 @@ func (r *RHEL) String() string {
 
 func (p *RHEL) GetBuildPackages() []string {
 	packages := []string{
-		"glibc", // ldconfig
+		"glibc",           // ldconfig
+		"platform-python", // osbuild
```

Comment:
I'm really sure this will work due to https://bugzilla.redhat.com/show_bug.cgi?id=2121775

Reply:
I realised this was missing because I tried to build all the manifests that changed and they were failing immediately.  Adding this was enough for RHEL 9.  I guess if we find that we need `python3` for RHEL 8 we can add that too.

Reply:
Interesting. I though (based on the bug report) that osbuild stages are actually not able to use `platform-python` due to hardcoding the binary path in the shebang. So installing it should not help much 🤔 I didn't investigate this more at all so I may be all wrong...

Reply:
The [RHEL 8.x runners](https://github.com/osbuild/osbuild/blob/f34bee944bdfc590cf25159521ee66c515d0cf10/runners/org.osbuild.rhel82#L1) use `platform-python`, so that's all we need in the build root.

I think the bug report is a case of removing the dependency from the host and not being able to bootstrap the build root.

Reply:
Thanks for the additional info. This is very helpful!

However I'm afraid that it is not completely true, since `python36` gets pulled into the buildroot pipeline and it is installed there. So while you are right that the bug is caused by the situation that you've described, AFAICT osbuild stages still use the `python3` installed in the buildroot, because they are executed in a separate process without an explicit interpreter path provided in the [`subprocess.run()` call](https://github.com/osbuild/osbuild/blob/f34bee944bdfc590cf25159521ee66c515d0cf10/runners/org.osbuild.rhel82#L76-L78).

Reply:
Our current RHEL 8 (and thus 9) runner also has an implicit dependency on `python3.6`:

https://github.com/osbuild/osbuild/blob/f34bee944bdfc590cf25159521ee66c515d0cf10/runners/org.osbuild.rhel82#L54-L63

Reply:
That's exactly the point that I'm trying to make... Therefore the buildroot packages should explicitly list `python36` and not rely on it being pulled in as a dependency of something else that is installed in the buildroot...

Reply:
See also https://bugzilla.redhat.com/show_bug.cgi?id=2121775

Reply:
So do we add `python36` to the runner deps for RHEL to be explicit?

Reply:
So it looks like it is pulled into the DNF transaction by some other package, so specifying it in the build packages will have basically no real effect.

Personally I suggest to add it there to make the dependency explicit and prevent any unexpected issues due to `python36` not being installed. It should be there until the rhel8 runner is "fixed" to depend only on `platform-python`.

Including a comment about the whole situation explaining two pythons being listed would be probably also good, but I'm pretty sure you would add something like this anyway 😉

Reply:
Done!